tests/integration-tests/tests/schedulers/test_custom_munge_key/test_custom_munge_key/pcluster.config.yaml

```diff
@@ -0,0 +1,37 @@
+Image:
+  Os: {{ os }}
+LoginNodes:
+  Pools:
+    - Name: pool
+      InstanceType: {{ instance }}
+      Count: 1
+      Networking:
+        SubnetIds:
+          - {{ public_subnet_id }}
+      GracetimePeriod: 3
+HeadNode:
+  InstanceType: {{ instance }}
+  Networking:
+    SubnetId: {{ public_subnet_id }}
+  Ssh:
+    KeyName: {{ key_name }}
+Scheduling:
+  Scheduler: slurm
+  SlurmSettings:
+    MungeKeySecretArn: {{ custom_munge_key_arn }}
+  SlurmQueues:
+    - Name: queue
+      Networking:
+        SubnetIds:
+          - {{ private_subnet_id }}
+      ComputeResources:
+        - Name: compute1
+          Instances:
+            - InstanceType: c4.xlarge
+          MinCount: 1
+          MaxCount: 5
+        - Name: compute2
+          Instances:
+            - InstanceType: c5.xlarge
+          MinCount: 1
+          MaxCount: 5
```

Comment:
Do we need these compute resources to be of type `c`? Can't we use `t2.medium`? Also, do they have to be two different compute resources?

Reply:
`t2.medium` is enough, I used `c` because I saw other yaml in integration tests using `c`. I have changed it to `t2.medium`.

Reply:
Also reduce the compute resources to 1.